index.html

```diff
@@ -10,11 +10,11 @@
             <li>ls -la: lists hidden flies</li>
         </ul>
     </li>
-    <li>cd</li>
+    <li>cd: change directory</li>
     <li>mkdir: Creates folder</li>
     <li>touch: create file</li>
-    <li>rm -Rf</li>
-    <li>pwd: </li>
+    <li>rm -Rf: remove program</li>
+    <li>pwd: Shows which file we are in</li>
```

Comment:
can you be INSIDE a file in terminal? Really it should say what directory we are in no?